integrationtest/ValidationProject/ValidateStrykerResults.cs

```diff
@@ -20,7 +20,7 @@ public void NetFullFramework()
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
                 var directory = new DirectoryInfo("../../../../TargetProjects/NetFramework/FullFrameworkApp.Test/StrykerOutput");
-                directory.GetFiles("*.json").ShouldNotBeNull("No reports available to assert");
+                directory.GetFiles("*.json", SearchOption.AllDirectories).ShouldNotBeEmpty("No reports available to assert");
```

Comment:
This construction was never working - GetFiles would always return empty here because:
a) it just [doesn't return null](https://learn.microsoft.com/en-us/dotnet/api/system.io.directory.getfiles?view=net-7.0#system-io-directory-getfiles(system-string))
b) it doesn't search recursively by default whereas the reports are beyond another level of folders (timestamps)